File: abc/abc132/abc132_c.cs
// https://atcoder.jp/contests/abc132/submissions/8272996

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        public static void Main()
        {
            var N = int.Parse(Console.ReadLine());
            var D = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();

            Array.Sort(D);

            Console.WriteLine(D[N / 2] - D[N / 2 - 1]);
        }
    }
}

File: abc/abc132/abc132_c.fs
// https://atcoder.jp/contests/abc132/submissions/11967782

let N = stdin.ReadLine() |> int

let d =
    stdin.ReadLine().Split()
    |> Array.map int
    |> Array.sort

d.[N / 2] - d.[N / 2 - 1] |> stdout.WriteLine
